superduperdb/backends/mongodb/query.py

```diff
@@ -394,9 +394,23 @@ def add_fold(self, fold: str):
     @applies_to('insert_many', 'insert_one')
     def documents(self):
         """Return the documents to insert."""
+
+        def _wrap_document(document):
```

Comment:
should we move it to base and use it in ibis and mongodb both
@jieguangzhou @blythed ?

Reply:
This is a general logic, can we move it into `query.__post_init__`